bitmap-steven-hawa-caleb/lib/bluescale-colors.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+const bmp = require(`${__dirname}/../model/bitmap-constructor.js`);
```

Comment:
@maschigokae @BatemanVO @abdih17 - no need to use `__dirname` for absolute paths when requiring in a custom module - for custom module `require` imports, just use the relative path